### ISSUE TITLE
[RPP Back Office] Add Permit Replacement Process

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#4061]
Build process for residents who need to replace permits that have been lost
- Residents usually call or email when a replacement permit is needed
How to determine the cost of a replacement permit:
- if they have purchased under 6 they will then pay for whatever the next additional permit cost is. 
For example if they have already purchased 2 permits, it will be just like they are paying for permit 3. 
If they already purchased 3 permits they will then be paying for permit 4 and so on.
In a case where they have already purchased the max of 6 and they then lose a permit, we will then be charging them the cost of permit # 6 which is $70 + tax = $75.78
- will need to determine how many permits have been purchased for a specific address in order to determine the cost of the replacement permit
- will need to track the old permit # issued and the new permit # issued

**Comment by ChrispinP** [comment]
Permit Replacement, Renewal, and Additional are now options in the application process. We used some JS to calculate costs for us instead of all the conditional rules and slow Knack equations. This has been tested and is now live in production. Due to using JS we were also able to eliminate unnecessary fields on the permit_application object and allow RPP staff to work with the same numbers the customer enters and the same number that finance, the cashier, and reports will use after migrating historical cost and number values. This also resulted in the Invoice becoming a more accurate display page of estimated costs rather than a final sale invoice and we removed dependent rules as a result. Staff should be able to see application type, update existing permit numbers, and be able to adjust costs accordingly.

complete, under review
